specification/examples/greeter.samt

```diff
@@ -15,14 +15,18 @@ record GreetResponse {
   message: String(pattern("a-z"))
 
   @Description("The precise time at which the response was generated")
+  @Description("foo bar
+    This is some very long comment
+    which describes the service.
+  baz")
   timestamp: DateTime
 }
 
 @Author("Leonard Schütz")
-@Description("
+@Description("foo bar
   This is some very long comment
   which describes the service.
-")
+baz")
```

Comment:
I personally like it more if the `greeter.samt` stays "realistic", but we could edit / move the `debug.samt` to include all kinds of edge-cases

Reply:
I agree